themes/V3/5ePHB/style.less

```diff
@@ -684,7 +684,7 @@ h5 + table{
 	columns    : 1;
 	text-align : center;
 	&:after {
-		all: unset;
+		display: none;
```

Comment:
Any particular reason for changing this? ` all:unset` completely removes the footer. `display:none` just hides it.

Reply:
Precisely, found users wanting to have a footer in pages that don't have one normally, this change makes easier to bring it back instead of having to redeclare every property. We use display none in the TOC page, so this helps with consistency too.

Reply:
Ok, that makes sense. Thanks!